### ISSUE TITLE
Syntax Highlighting for Mustache IF

### ISSUE THREAD

**Issue by dwsmyth** [danwhite85/vscode-mustache-syntax-highlighting#4]
Hi Dan,
I am a big fan of logicless templates and mustache, so when I saw your syntax highlighting I jumped on it. I am having some difficulty with the syntax highlighting in Dark themes.

Here is a simple example of how using Mustache tags in a HTML document stops syntax highlighting.
`<input type="text" {{#is_read_only}}disabled{{/is_read_only}} name="test" value=""/>`

In the Dark color schemes `/is_read_only}}` is highlighted in one color, but `{{^is_read_only}}` is in another.

**Comment by dnwhte** [comment]
I don't believe there is much I can do about this. It appears that vscode thinks there is a syntax error.

It adds the class mtk10 to the segment "{{/is_read_only}}" when it's in an html tag. 

I see some have started adding their own stylesheets to override some of that stuff:
https://gist.github.com/DoroGi/56d6091e4bfc30c933ce697ebf1649d4

If you're interested here are the vscode tests:
https://github.com/Microsoft/vscode/blob/4c8e61c0499dfd7531ec99ab6723c1e4371ed2f3/src/vs/editor/test/common/viewLayout/viewLineRenderer.test.ts

I could be wrong, but to the best of my knowledge that's what is going on.

**Comment by michaeljsalo** [comment]
I am also using with VSCode dark themes, and can't tell if syntax highlighting is working right.  Are open tags and close tags meant to show as different colors?
<img width="270" alt="Screen Shot 2019-08-05 at 4 18 00 PM" src="https://user-images.githubusercontent.com/1945279/62495830-b1f64a00-b79c-11e9-8f1d-f8f726484f4a.png">
